.gitignore

```
@@ -146,7 +146,7 @@ Desktop.ini
 .dbus/
 repos/
 repos-download/
-compass/
+/compass/
```

```suggestion
/compass
```
I suggest to revert this to the previous version.